addon/services/raven.js

```diff
@@ -4,7 +4,7 @@ export default RavenService.extend({
     releaseMetaName: 'sentry:revision',
     release: Ember.computed({
         get: function() {
-            return $("meta[name='revision']").attr('content');
+            return $("meta[name='sentry:revision']").attr('content');
```

Comment:
You are right, that this is wrong ... ;-)

But doesn't look like the perfect fix, afaict the code was supposed to use the releaseMetaName property. Could you make the release property depend on releaseMetaName and use that as name in the jquery selector?